client/store/races.js

```diff
@@ -0,0 +1,126 @@
+import axios from 'axios'
+
+/**
+ * ACTION TYPES
+ */
+const GET_RACES = 'GET_RACES'
+const SELECT_RACE = 'SELECT_RACE'
+const ADD_RACE = 'ADD_RACE'
+const EDIT_RACE = 'EDIT_RACE'
+
+/**
+ * INITIAL STATE
+ */
+const defaultRaceState = {
+  allRaces: [],
+  selectedRace: {}
+}
+
+/**
+ * ACTION CREATORS
+ */
+export const gotRaces = races => ({
+  type: GET_RACES,
+  races
```

Comment:
i think the variable should be the same name as the item in the state that it is replacing, so if it is called allRaces in defaultRaceState, it should be called allRaces in the action creators